src/postamble.js

```diff
@@ -158,6 +158,10 @@ function ExitStatus(status) {
 
 var calledMain = false;
 
+#if STANDALONE_WASM && MAIN_READS_PARAMS
+var mainArgs = undefined;
```

Comment:
this `var` could be lower down, where we have `mainArgs = ` without a var?

but this variable is only written to, so I'm not sure what it does, it looks like dead code atm?

Reply:
This needs to be global var so that src/library_wasi.js can access it.